polaris-react/src/components/Avatar/Avatar.tsx

```diff
@@ -1,19 +1,13 @@
 import React, {useState, useCallback, useEffect} from 'react';
 
-import type {Experimental} from '../../types';
 import {classNames, variationName} from '../../utilities/css';
 import {useI18n} from '../../utilities/i18n';
 import {useIsAfterInitialMount} from '../../utilities/use-is-after-initial-mount';
 import {Image} from '../Image';
 
 import styles from './Avatar.scss';
 
-type Size =
-  | 'extraSmall'
-  | 'small'
-  | 'medium'
-  | 'large'
-  | Experimental<'xl' | '2xl'>;
+type Size = 'xs' | 'small' | 'medium' | 'large' | 'xl' | '2xl';
```

Comment:
How did we determine these size names? These are half verbose half tshirt sizes which seems weird to me. Are we doing this somewhere else? Should we match breakpoint naming conventions?

```suggestion
type Size = 'xs' | 'sm' | 'md' | 'lg' | 'xl' | '2xl';
```

Reply:
Excellent, agreed! I will revise to match the breakpoints then. Thanks @sarahill @kyledurand